classes/statistics/TemporaryItemInvestigationsDAO.php

```diff
@@ -29,7 +29,10 @@ class TemporaryItemInvestigationsDAO extends PKPTemporaryItemInvestigationsDAO
      */
     protected function getInsertData(object $entryData): array
     {
-        return ['chapter_id' => $entryData->chapterId];
+        return array_merge(
```

Comment:
Hmm, I've seen this before (perhaps it was me who removed the `parent::getInsertData()`).

Here I didn't find a `getInsertData()` on the parent class.

Reply:
I just saw that you've added this method on the other PR: https://github.com/pkp/pkp-lib/pull/9629/files#diff-bb4c754d4ff0dcfb6466d43ecaa5f24aa8e177d0a1ad9d6cd6136dd95d95722cR47 😁